src/lib.rs

```diff
@@ -52,8 +50,8 @@ pub enum LogLevel {
     Warning,
 }
 
-fn from_byte_string(byte_str: *const c_char, length: size_t) -> Result<String, Utf8Error> {
-    let mut vec_str: Vec<u8> = Vec::with_capacity(length as usize);
+fn from_byte_string(byte_str: *const c_char, length: usize) -> Result<String, Utf8Error> {
```

Comment:
So we can assume `size_t` is `usize` (except for niche platforms, which will break as explained [here](https://github.com/rust-lang/rust-bindgen/issues/1903) and [here](https://github.com/dtolnay/cxx/pull/576/files#diff-5e36084aec44d61b84d68bf00fcf75d7164d22496225c357d8f3a56b5f4768b5R261))